src/tools/compiletest/src/runtest.rs

```diff
@@ -31,12 +28,27 @@ use crate::compute_diff::{write_diff, write_filtered_diff};
 use crate::errors::{self, Error, ErrorKind};
 use crate::header::TestProps;
 use crate::read2::{read2_abbreviated, Truncated};
-use crate::util::{add_dylib_path, copy_dir_all, dylib_env_var, logv, static_regex, PathBufExt};
-use crate::{extract_gdb_version, is_android_gdb_target, json, ColorConfig};
+use crate::util::{add_dylib_path, logv, static_regex, PathBufExt};
+use crate::{json, ColorConfig};
 
+// tidy-alphabet-start
+mod assembly;
+mod codegen;
+mod codegen_units;
 mod coverage;
+mod crash;
 mod debugger;
-use debugger::DebuggerCommands;
```

Comment:
I should move `debugger` out of this listing because it's a helper module not a test suite/mode. I should also add a comment to say the intended purpose of the organization of helper submodules as each implementing a test suite/mode.

Reply:
Fixed.